test_regress/t/t_flag_build_cmake.pl

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2008 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+
+scenarios(simulator => 1);
+top_filename("t/t_flag_make_cmake.v");
+
+compile(  # Don't call cmake nor gmake from driver.pl
+    verilator_make_cmake => 0,
+    verilator_make_gmake => 0,
+    verilator_flags2 => ['--exe --cc --build -j 2 --make cmake',
+        '../' . $Self->{main_filename},
+        ' -MAKEFLAGS "-DTEST_NAME=' . $Self->{name} . '"',
+        ' -MAKEFLAGS "-DTEST_CSOURCES=' . $Self->{main_filename} . '"',
```

Comment:
Please check the test will fail if MAKEFLAGS don't work (are just ignored).  It might check this but wasn't obvious.

Reply:
Now the test fails if MAKEFLAGS is not passed properly. (t_flag_make.pl does similar thing too for gmake)
https://github.com/verilator/verilator/blob/576c4a83401dd70fdaa7a0ff4f540eebddf06a3e/test_regress/t/t_flag_build_cmake.pl#L37-L39